tracing-opentelemetry/src/subscriber.rs

```diff
@@ -571,17 +571,29 @@ where
 
                 if self.event_location {
                     let builder_attrs = builder.attributes.get_or_insert(Vec::new());
-                    if let Some(file) = meta.file() {
-                        #[cfg(feature = "tracing-log")]
-                        builder_attrs.push(KeyValue::new("code.filepath", file.to_owned()));
-                        #[cfg(not(feature = "tracing-log"))]
-                        builder_attrs.push(KeyValue::new("code.filepath", Cow::Borrowed(file)));
+
+                    #[cfg(feature = "tracing-log")]
+                    let (file, module) = match &normalized_meta {
+                        Some(meta) => (
+                            meta.file().map(|s| Value::from(s.to_owned())),
+                            meta.module_path().map(|s| Value::from(s.to_owned())),
+                        ),
+                        None => (
+                            event.metadata().file().map(|s| Value::from(s)),
```

Comment:
clippy wishes this was
```suggestion
                            event.metadata().file().map(Value::from),
```